__tests__/posts.ts

```diff
@@ -889,7 +890,7 @@ describe('query postUpvotes', () => {
     postUpvotes(id: $id) {
       edges {
         node {
-          createdAt
+          createdAt: votedAt
```

Comment:
This change will need to be done on `POST_UPVOTES_BY_ID_QUERY` on the clients to preserve interfaces. This is because in `UserPost` the `votedAt` field is the actual time of voting vs `createdAt` that is just insertion date for the row.

Reply:
Tested it, we just need to adjust `Upvote` interface for `votedAt` instead of `createdAt`.

Reply:
Adjusted here https://github.com/dailydotdev/apps/pull/2063